### PR TITLE
Wok 621 xslt to jsonld

### PR DESCRIPTION
**Pull request by VictorHarbo**
This PR contains XSLT that transforms MODS to Schema.org JSON-LD.

### CODE REVIEW

File: src/test/java/dk/kb/present/transform/XSLTSchemaDotOrgTransformerTest.java [hunk 1]
```diff
@@ -63,16 +69,116 @@ void testCreatorsAndHeadline() throws Exception {
...
+
+        String correctString = importTestFile("src/test/resources/schemaOrgJsonTestFiles/schemaOrg_40221e30-1414-11e9-8fb8-00505688346e.json");
+        Assertions.assertEquals(schemaOrgString, correctString);
+
+    }
+
+
+    /**
+     * Update test files when XSLT changes.
+     */
```
Comment (tokee) [inline src/test/java/dk/kb/present/transform/XSLTSchemaDotOrgTransformerTest.java:156]: I understand what is happening here. It is useful and at the same time a bit temptation to just regenerate the test files without inspecting them to see if the change was benign.

Let's discuss this face2face.

**Review by tokee — CHANGES_REQUESTED** [review]
Only one real issue: Injection of the imageserver.

**Comment by VictorHarbo** [comment]
Testfiles and XSLT has been updated and your generalized parameter injection has been merged. There are some old unittest files present, that are to be deleted with the cleanup being done in branch WOK_640

File: src/test/resources/schemaOrgJsonTestFiles/schemaOrg_40221e30-1414-11e9-8fb8-00505688346e.json [hunk 2]
```diff
@@ -1 +1 @@
...
...e":"1786-9-18","deathDate":"1848-1-20","hasOccupation":{"@type":"Occupation","name":"konge af danmark (1839-1848)"}},"F. 1885"],"keywords":["Danske portrætter","Danmarks regenter","Christian 08"],"dateCreated":"1848","temporal":"Created between 1820 and 1848","publication":{"@type":"PublicationEvent","description":["1820","1848"]},"identifier":[{"@type":"PropertyValue","PropertyID":"UUID","value":"40221e30-1414-11e9-8fb8-00505688346e"},{"@type":"PropertyValue","PropertyID":"ShelfLocator","value":"Billedsamlingen. Danske portrætter, 2°, Christian 8 (1786-1848), 11"},{"@type":"PropertyValue","PropertyID":"localIdentifier","value":"KE070592.tif"},{"@type":"PropertyValue","PropertyID":"accessionNumber","value":"1911-14241"}],"isPartOf":[{"@type":"Collection","headline":"Billedsamlingen","description":"Det Kgl. Biblioteks billedsamling","url":"https:\/\/www.kb.dk\/find-materiale\/samlinger\/billedsamlingen\/index.html"}],"image":{"@type":"ImageObject","contentSize":"152932120","contentURL":"https:\/\/example.com\/imageserver\/\/DAMJP2\/DAM\/Samlingsbilleder\/0000\/624\/420\/KE070592","height":"8141","width":"6261","dateCreated":"2018-12-12T13:36:42.000+01:00"},"kb:read_direction":"LTR"}
```
Comment (tokee) [inline src/test/resources/schemaOrgJsonTestFiles/schemaOrg_40221e30-1414-11e9-8fb8-00505688346e.json:1]: The image URL has a double slash: https://example.com/imageserver//DAMJP2/DAM/Samlingsbilleder/0000/624/420/KE070592 . Technically it works, but it ain't pretty. I know it's my fault for putting the slash in the properties, but maybe remove it from the properties and add a comment saying it should not end in slash? Or the fancy version: Remove the slash in the XSLT if it is present?

**Review by tokee — CHANGES_REQUESTED** [review]
Grasping at straws here by asking for nicer URLs.

File: conf/ds-present-behaviour.yaml [hunk 3]
```diff
@@ -131,5 +131,5 @@ config:
                     stylesheet: 'xslt/mods2solr.xsl'
 
   # Imageserver to be used when constructing URLs to be shown in Schema.org XSLT transformation
-  # This has to be set before image URLs are working.
+  # This has to be set before image URLs are working. imageserver has to pe set without trailing slash as this is currently handled by the XSLT transformation
```
Comment (tokee) [inline conf/ds-present-behaviour.yaml:134]: Plese mirror the `p` vertically.

**Review by tokee — CHANGES_REQUESTED** [review]
Typo!

**Review by tokee — APPROVED** [review]

**Pull request merged**